inc/uart_i2c.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2020  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+#ifndef UART_I2C_H
+#define UART_I2C_H
+
+void uart_send(uint8_t* data, uint32_t length);
```

Comment:
I changed the mock functions from telecom and telemetry functions to uart & i2c functions which we can implement driver functions later here. This helped me to implement the frame structure which allows for deeper testing.